Assets/HoloToolkit/UX/Scripts/Keyboard.cs

```diff
@@ -901,5 +1018,42 @@ private void ResetKeyboardState()
         }
 
         #endregion Keyboard Layout Modes
+
+        /// <summary>
+        /// Respond to keyboard activity: reset timeout timer, play sound
+        /// </summary>
+        private void IndicateActivty()
+        {
+            ResetClosingTime();
+            if (_audioSource == null)
+            {
+                _audioSource = GetComponent<AudioSource>();
```

Comment:
We should probably add the class attribute that requires the AudioSource component.

Reply:
I do not agree. The audio source could be attached to the keyboard and spatialized so clicking actually comes *FROM* the keyboard. That is how I use it.